src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.scss

```diff
@@ -0,0 +1,189 @@
+@import 'mixins';
+@import 'colors';
+
+
+::ng-deep .mat-mdc-menu-panel {
```

Comment:
I think we need to prefix this with `:host`  to avoid this leaking outside this component

Reply:
I definitely tried that initially, but these styles break if they're inside of :host, at least in storybook, and I'm not sure why-I suspect it might be because they're in the CDKLayer, so I'm not sure what options we have in that case.

Reply:
got it! I think you can prefix them by adding a class to the menu

Reply:
I was able to get this to work with an added class on mat-menu, but I'm not sure if there's a better way to do this. `::ng-deep` is dark magic.

Reply:
well but `.mat-mdc-menu-panel` will still affect all menu panels. It should be a class you added so it only affects this one